core/embed/rust/src/ui/layout_caesar/component/input_methods/choice.rs

```diff
@@ -117,6 +127,7 @@ where
             page_counter: 0,
             items_distance: DEFAULT_ITEMS_DISTANCE,
             is_carousel: false,
+            is_cancelable: false,
```

Comment:
wondering: what would happen if we used `!is_carousel` instead of `is_cancelable` ?

i'm thinking _most_ places should be cancellable, and the only place comes to mind where that is undesirable is `select_word` ... where we can cheat and make it a carousel :)

Reply:
I could try that, but then I would drop the `is_carousel` flag rather than the `is_cancelable` flag because ... it's strange to change the cancelability based on `is_carousel`, but rendering it as non-carousel whenever it is cancelable seems less strange given there is no way to show the cancel button on carousel mode...

Reply:
how about:
```rust
enum ChoiceControls {
    Carousel,
    Cancellable,
}
```

Reply:
Still an experiment, but it might work: 812cb8d77f656e5b7b9a37c4bf9f423f6543b63d